runtime/src/lib.rs

```diff
@@ -466,6 +466,11 @@ impl<R: OnUnbalanced<NegativeImbalance>> OnUnbalanced<NegativeImbalance> for Dea
     }
 }
 
+pub struct DealWithFeesPoA<R>(PhantomData<R>);
```

Comment:
`DealWithFeesPoA` doesn't seem to be the right name, how about `BurnAllTxFees` and lets add a test similar to the one for `DealWithFees`. Arguably that should also be renamed to reflect how it handles fees (now that we have two implementations)

Reply:
Also since for this impl we are not sending fees to anywhere it doesn't really need to be generic. So it can be simply `pub struct BurnAllTxFees {}`